### ISSUE TITLE
Reuploading exported logs

### ISSUE THREAD

**Issue by nghildebrandt** [tensorflow/tensorboard#4413]
I have export logs that I uploaded. Since they are exported to json files, they are not in the same format as they have been uploaded, so it would be nice with a tool or way to convert the exported into a  format (Summary proto) that can be re-uploaded to tensorboard

**Comment by bileschi** [comment]
Agreed.  Right now the uploader only accepts the Summary format. It would be nice to also accept the format that the downloader gives.   Let's consider this a feature request for TensorBoard.dev
